src/main/scala/firrtl/passes/AnnotateMemMacros.scala

```diff
@@ -21,35 +21,35 @@ case class AppendableInfo(fields: Map[String, Any]) extends Info {
 }
 
 object AnalysisUtils {
-
-  def getConnects(m: Module) = {
-    val connects = mutable.HashMap[String, Expression]()
-    def getConnects(s: Statement): Statement = {
-      s map getConnects match {
+  type Connects = collection.mutable.HashMap[String, Expression]
```

Comment:
and therefore it's better to leave everything as mutable until some better way to do this is figured out?